README.md

```diff
@@ -1,33 +1,103 @@
 # overseas-entities-web
-Web front-end for the Register An Overseas Entity service
 
-### Requirements
+Web front-end for the **register an overseas entity and tell us about its beneficial owners** service. Link to the live page [here](https://www.gov.uk/guidance/register-an-overseas-entity)
+</br>
+
+## Overseas entities architecture
+
+![Overseas entities architecture](./docs/Overseas%20entities%20architecture.png)
```

Comment:
on the diagram, it should go API -> filing-resource-handler - chips-filing-consumer